proto/contlcycle/contlcycle.proto

```diff
@@ -44,4 +44,7 @@ message ContainerEvent {
 message PodEvent {
     string podUID = 1; // podUID contains the pod UID
     string source = 2; // source contains the pod source (kubelet)
+    oneof optionalExitTimestamp {
+        int64 exitTimestamp = 3; // exitTimestamp contains the exit timestamp of the pod
+    }
```

Comment:
For optional fields, I think you can simply do:

```suggestion
    optional int64 exitTimestamp = 3; // exitTimestamp contains the exit timestamp of the pod
```

`oneof` is used for exclusive fields (like `union` in `C`)

Reply:
```proto/contlcycle/contlcycle.proto:47:14: Explicit 'optional' labels are disallowed in the Proto3 syntax. To define 'optional' fields in Proto3, simply remove the 'optional' label, as fields are 'optional' by default.```

Guess `optional` isn't needed at all

Reply:
Annnnnnddddddddd...without optional, what is generated is an int64, which is VERY MUCH not optional

Reply:
Indeed, it seems that this message was using `oneof` for other optional fields.
This was most probably a workaround for a limitation enforced by the `gogo` protobuf generator and that you described above when you tried `optional`.
The thing is that [`gogo` is deprecated](https://github.com/gogo/protobuf#readme) and not maintained anymore.
`contlcycle` message has been regenerated with the official and supported Google generator in #239.